web/crux-ui/e2e/utils/node-helper.ts

```diff
@@ -65,6 +65,13 @@ export const deployWithDagent = async (
   await navigation
 
   if (!ignoreResult) {
+    await page.waitForTimeout(1000)
+    await page.screenshot({ path: screenshotPath('should-successful-after-1k'), fullPage: true })
```

Comment:
Instead of `...-1k` we have to use `...-1s`, so the reader easily can identify the seconds for it, IMHO

Reply:
These are temporary stuff. Seems we can finally strike off the last flaky test. We're working on it with @m8vago. After I clean up, please remind me to remove these lines if I forget to.

Reply:
Resolved.